livekit-ffi/protocol/room.proto

```diff
@@ -144,6 +144,28 @@ message SetLocalMetadataCallback {
   optional string error = 2;
 }
 
+message SendChatMessageRequest {
+  uint64 local_participant_handle = 1;
+  string message = 2;
+  repeated string destination_identities = 3;
+  optional string sender_identity = 4;
+}
+message EditChatMessageRequest {
+  uint64 local_participant_handle = 1;
+  string edit_text = 2;
+  ChatMessage original_message = 3;
```

Comment:
Do we need the whole ChatMessage? Is it possible to just use the message_id?

Reply:
in an ideal world, yeah, the ID would be enough. 
We don't have a central authority for chat messages though. So for an edited message we'd still want to keep the e.g. original timestamp of the message, so passing the whole message seemed like the best idea. 
But would be happy about alternative suggestions